crates/red_knot/README.md

```diff
@@ -0,0 +1,27 @@
+# Red Knot
+
+Red Knot is an extremely fast type checker.
+Currently, it is a work-in-progress and not ready for user testing.
+
+Red Knot is designed to prioritize good type inference, even in unannotated code,
+and aims to avoid false positives.
+It will *not* be a drop-in replacement for either Mypy or Pyright.
```

Comment:
While this is true, I worry that this might be the wrong emphasis. We certainly won't be a drop-in replacement for either tool in every respect, but it's important to us that it's not too onerous for users to switch from one of those tools to red-knot

Reply:
Probably something like

```
While red-knot will produce similar results to mypy and pyright on many codebases,
100% compatibility with these tools is a non-goal. On some codebases, red-knot's design decisions
lead to different outcomes than you would get from running one of these more established tools.
```